CI is flaky

I saw several times failing tests with "Operation not permitted" errors while starting the ceph cluster. Re-running the test usually succeeds.
```
310 ++ ceph-osd --id 0 --mkjournal --mkfs
311 2020-12-21 15:15:27.677 7fa2ae6f2a80 -1 memstore(/tmp/ceph/osd) /tmp/ceph/osd
312 ++ ceph-osd --id 0
313 2020-12-21 15:15:27.781 7f48a3c19a80 -1 Falling back to public interface
314 2020-12-21 15:15:27.801 7f48a3c19a80 -1 osd.0 0 log_to_monitors {default=true}
315 2020-12-21 15:15:27.805 7f48a3c19a80 -1 osd.0 0 mon_cmd_maybe_osd_create fail: 'wrong fsid': (1) Operation not permitted
316 2020-12-21 15:15:27.805 7f48a3c19a80 -1 osd.0 0 init unable to update_crush_device_class: (1) Operation not permitted
317 make: *** [test-container] Error 250
318 Makefile:43: recipe for target 'test-container' failed
319 Error: Process completed with exit code 2.
```

See for example here: https://github.com/ceph/go-ceph/pull/419/checks?check_run_id=1589665458

--- a/micro-osd.sh
+++ b/micro-osd.sh
@@ -77,7 +77,8 @@ ceph-mon --id ${MON_NAME}
 # start an osd
 OSD_ID=$(ceph osd create)
 ceph osd crush add osd.${OSD_ID} 1 root=default host=localhost
-ceph-osd --id ${OSD_ID} --mkjournal --mkfs
+ceph-osd --id ${OSD_ID} --mkjournal --mkfs --foreground
+sync # this is an attempt to fix CI issue #423, remove if it has no effect
 ceph-osd --id ${OSD_ID}
 
 # start an mds for cephfs

